pkg/minikube/machine/cache_images.go

```
@@ -329,6 +329,11 @@ func removeExistingImage(r cruntime.Manager, src string, imgName string) error {
 	}
 
 	errStr := strings.ToLower(err.Error())
+	if strings.Contains(errStr, "is using its referenced image") {
+		klog.Warningf("Image %s is being used by a running container, skipping deletion, it would be replaced later", imgName)
+		return nil
```

can you explain why would we wanna return nil ? and not return an error ?

---

the upstream error is throw by `docker image rm`, but docker will failed to untag the image if there is a running container using this image.

but that's fine, the next steps would load image and replace the tag with new image.

so we could ignore this error safely, also complete the origin purpose:

- `minikube image load` should replace the old existing image